sixtyfps_compiler/generator/rust.rs

```diff
@@ -1247,7 +1247,13 @@ fn compile_expression(expr: &Expression, component: &Rc<Component>) -> TokenStre
                     if let Expression::ElementReference(focus_item) = &arguments[0] {
                         let focus_item = focus_item.upgrade().unwrap();
                         let focus_item = focus_item.borrow();
-                        let item_index = focus_item.item_index.get().unwrap();
+                        let item_index = if focus_item.repeated.is_some() {
+                            // We want the repeated instance itself; _self already points to it, and the root element
```

Comment:
i'm concern that _self does not always points to it as it could be a deeper instance.

Reply:
You're right, `self` is incorrect in this nested case.